src/Tribe/PUE/Package_Handler.php

```diff
@@ -70,9 +70,9 @@ public function filter_upgrader_pre_download( $reply, $package, WP_Upgrader $upg
 	 * @return bool
 	 */
 	protected function is_mt_package( $package ) {
-		if ( empty( $package )
-		     || ! preg_match( '!^(http|https|ftp)://!i', $package )
-		     || file_exists( $package )
```

Comment:
Well...you can actually ignore me completely. I totally missed the `!` preceding the `preg_match( '!^(http|https|ftp)://!i', $package )`. I read that as: empty || is URL || file exists.

Sorry to make you type all of that - though it was a good read :)